src/components/App.jsx

```diff
@@ -179,12 +180,16 @@ class App extends Component {
 		}
 	}
 
-	logout() {
-		this.props.logout();
-		browserHistory.push("/");
-		this.setState({
-			profileMenuOpen: false
-		});
+	handleKeycloakLogout() {
+		const redirectUri = `${location.origin}/`
+		try {
+			this.props.logout();
+			keycloak.logout({
+				redirectUri: redirectUri
+			});
+		} catch (error) {
+			console.log("Logout error", error);
```

Comment:
Recently I start to realize that there is `console.error ` might be better to document error